iis-wf/iis-wf-affmatching/src/main/java/eu/dnetlib/iis/wf/affmatching/AffMatchingJob.java

```diff
@@ -90,12 +101,37 @@ private static AffMatchingService createAffMatchingService() {
         affMatchingService.setAffiliationReader(new IisAffiliationReader());
         affMatchingService.setOrganizationReader(new IisOrganizationReader());
         
+        DocumentOrganizationReader documentOrganizationReader = new DocumentOrganizationReader();
+        documentOrganizationReader.setDocumentProjectReader(new IisDocumentProjectReader());
+        documentOrganizationReader.setProjectOrganizationReader(new IisProjectOrganizationReader());
+        documentOrganizationReader.setDocumentOrganizationCombiner(new DocumentOrganizationCombiner());
+        documentOrganizationReader.setDocProjConfidenceLevelThreshold(params.inputDocProjConfidenceThreshold);
+        
+        affMatchingService.setDocumentOrganizationReader(documentOrganizationReader);
+        
         
         // writer
         
         affMatchingService.setAffMatchResultWriter(new IisAffMatchResultWriter());
         
         
+        // docOrgRelationAffOrgMatcher
+        
+        DocOrgRelationAffOrgJoiner docOrgRelationAffOrgJoiner = new DocOrgRelationAffOrgJoiner();
+        
+        AffOrgMatchComputer docOrgRelationAffOrgMatchComputer = new AffOrgMatchComputer();
+        docOrgRelationAffOrgMatchComputer.setAffOrgMatchVoters(ImmutableList.of(
+                createNameCountryStrictMatchVoter(),
+                createNameStrictCountryLooseMatchVoter(),
+                createSectionedNameStrictCountryLooseMatchVoter(),
+                createSectionedNameLevenshteinCountryLooseMatchVoter(),
+                createSectionedShortNameStrictCountryLooseMatchVoter()));
```

Comment:
I'm not sure how dynamic this configuration would become but I can imagine Spring simplifying voters configuration as well.